ra-optimisation/search-tree-rao/src/main/java/com/powsybl/openrao/searchtreerao/linearoptimisation/algorithms/fillers/MaxMinRelativeMarginFiller.java

```diff
@@ -52,25 +52,18 @@ public MaxMinRelativeMarginFiller(Set<FlowCnec> optimizedCnecs,
     }
 
     @Override
-    public void fill(LinearProblem linearProblem, FlowResult flowResult, SensitivityResult sensitivityResult) {
-        super.fill(linearProblem, flowResult, sensitivityResult);
+    public void fill(LinearProblem linearProblem, FlowResult flowResult, SensitivityResult sensitivityResult, RangeActionActivationResult rangeActionActivationResult) {
+        super.fill(linearProblem, flowResult, sensitivityResult, rangeActionActivationResult);
         buildMinimumRelativeMarginSignBinaryVariable(linearProblem);
         updateMinimumNegativeMarginDefinition(linearProblem);
         Set<FlowCnec> validFlowCnecs = FillersUtil.getFlowCnecsComputationStatusOk(optimizedCnecs, sensitivityResult);
         buildMinimumRelativeMarginVariable(linearProblem, validFlowCnecs);
-        buildMinimumRelativeMarginConstraints(linearProblem, validFlowCnecs);
+        // TODO : we may argue that "ptdfApproximationLevel.shouldUpdatePtdfWithPstChange()=false" has no real advantage anymore
```

Comment:
We can still avoid computing the zone to zone ptdfs (is the sensitivities of the glsks on the cnecs) during the sensitivity computation with the parameter set to false.